packages/nextjs/app/builders/0x59a04ed7D9Eb8c7B1A7958678D36f1dC2a13E096/page.tsx

```diff
@@ -0,0 +1,50 @@
+import { NextPage } from "next";
+
+const EnviloggerProfile: NextPage = () => {
+  return (
+    <figure className="md:flex bg-blue-100 rounded-xl p-8 my-8 max-w-4xl md:mx-auto mx-4">
+      {/* eslint-disable-next-line @next/next/no-img-element */}
+      <img
+        className="w-48 h-48 md:rounded-full md:mx-8 mx-auto"
+        src="https://avatars.githubusercontent.com/u/1270793"
+        alt="Anton Kudriavtsev"
+      />
+      <div className="pt-6 text-center md:text-left">
+        <h2 className="text-2xl leading-6 font-bold my-0 md:text-left">Anton Kudriavtsev</h2>
+        <span className="text-xs">📍 Budva, Montenegro</span>
+        <p className="text-gray-600">
+          Working as team lead @{" "}
+          <a className="link" href="https://naga.com/" target="_blank" rel="noopener noreferrer">
+            NAGA
+          </a>{" "}
+          with overall experience in software development almost 20 years. In early 2024 started learning smart contract
+          development and found{" "}
+          <a className="link" href="https://buidlguidl.com/">
+            BuidlGuidl
+          </a>{" "}
+          and{" "}
+          <a className="link" href="https://speedrunethereum.com/">
+            speedrunethereum.org
```

Comment:
The link is correct, but not the label :D Should be `.com`

Reply:
Oh yeah, thanks for noticing! It is fine now.